infra/Pos/Slotting/Util.hs

```diff
@@ -88,9 +72,21 @@ getSlotStartEmpatically
 getSlotStartEmpatically slot =
     getSlotStart slot >>= maybeThrow (SEUnknownSlotStart slot)
 
+-- | Get last epoch empatically, which means
+-- that function throws exception when last epoch is unknown.
+getLastEpochEmpatically
```

Comment:
This function should use `getLastEpoch` and then it will not need to throw exception.